sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/TextAnalyticsAsyncClient.java

```diff
@@ -572,13 +574,13 @@ public String getDefaultLanguage() {
      * @param language The 2 letter ISO 639-1 representation of language for the text. If not set, uses "en" for
      * English as default.
      *
-     * @return A {@link TextAnalyticsPagedFlux} contains a list of
-     * {@link LinkedEntity recognized linked entities document result}.
+     * @return A {@link Mono} contains a {@link TextAnalyticsResultCollection} of
+     * {@link RecognizeLinkedEntitiesResult recognized linked entities document result}.
      *
      * @throws NullPointerException if {@code documents} is {@code null}.
      */
-    @ServiceMethod(returns = ReturnType.COLLECTION)
-    public TextAnalyticsPagedFlux<RecognizeLinkedEntitiesResult> recognizeLinkedEntitiesBatch(
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<TextAnalyticsResultCollection<RecognizeLinkedEntitiesResult>> recognizeLinkedEntitiesBatch(
```

Comment:
Can this api be removed in favor of `recognizeLinkedEntitiesBatch(Iterable<String> documents, String language, TextAnalyticsRequestOptions options)`? Similarly, remove this overload for other operations too.

Reply:
Yes. We will keep one API which described at the updated PR's description.

Reply:
removed